lib/splitWhen/index.js

```diff
@@ -0,0 +1,10 @@
+import _curry2 from '../_internal/_curry2'
+
+export default _curry2(function splitWhen(cb, arr) {
+  var left = []
+  while (left.length < arr.length) {
+    if (cb(arr[left.length])) break
+    left.push(arr[left.length])
+  }
+  return [left, arr.slice(left.length)]
```

Comment:
maybe better have an index to increment and use instead of `left.length`?

Reply:
Isn't this shorter?

Reply:
@Kelin2025 I doubt if it is